server/command.go

```diff
@@ -143,6 +154,17 @@ func getAutoCompleteData() *model.AutocompleteData {
 	subscribe := model.NewAutocompleteData("subscribe", "", "Subscribe the current channel to notifications from Confluence")
 	confluence.AddCommand(subscribe)
 
+	config := model.NewAutocompleteData("config", "", "Config related options for confluence instances")
+	configOptions := []model.AutocompleteListItem{{
+		HelpText: "Add config for the confluence instance",
+		Item:     "add",
+	}, {
+		HelpText: "List all the saved configs",
```

Comment:
```suggestion
		HelpText: "List saved config",
```
This sounds better maybe?